google-cloud-storage/src/test/java/com/google/cloud/storage/conformance/retry/CtxFunctions.java

```diff
@@ -97,11 +98,16 @@
           Blob resolvedBlob = ctx.getStorage().create(blobInfo);
           return ctx.map(s -> s.with(resolvedBlob));
         };
-    private static final CtxFunction serviceAccount =
+    static final CtxFunction serviceAccount =
         (ctx, c) ->
             ctx.map(s -> s.with(ServiceAccount.of(c.getServiceAccountSigner().getAccount())));
-    private static final CtxFunction hmacKey =
-        (ctx, c) -> ctx.map(s -> s.with(ctx.getStorage().createHmacKey(s.getServiceAccount())));
+    static final CtxFunction hmacKey =
+        (ctx, c) ->
+            ctx.map(
+                s -> {
+                  HmacKey hmacKey1 = ctx.getStorage().createHmacKey(s.getServiceAccount());
```

Comment:
Not a big deal either way but generally the service account returned by `getServiceAccount` is the default GCS service account and not one you would use to create HMAC keys-- that said it doesn't matter either way to the emulator and you could even use a dummy email and it would be fine.

Reply:
In this case, the `getServiceAccount()` is against the test state, so it's a service account that was built as part of setup based on a canned email address.

Reply:
Ah, got it!